daisy_workflows/image_import/inspection/tests/test-data/images/gcp-image-sles-15-sp2-sap-v20200721.yaml

```diff
@@ -13,3 +13,78 @@ files:
     ID_LIKE="suse"
     ANSI_COLOR="0;32"
     CPE_NAME="cpe:/o:suse:sles:15:sp2"
+  /etc/products.d/SLES_SAP.prod: |
+    <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These XMLs are very verbose. Is this all necessary for inspection tests? E.g. how do languages affect inspection?

Reply:
Updated to include a single line -- for this inspection, we're only concerned with the existence of the file, not the content.